src/functions-reference/void_functions.qmd

```diff
@@ -63,3 +61,27 @@ U+000A; C++ literal `'\n'`) is inserted at the end of the printed
 line. The types `T1` through `TN` can be any of Stan's built-in
 numerical types or double quoted strings of characters (bytes).
 {{< since 2.18 >}}
+
+
+## Fatal error statement
+
+The fatal_error statement has the same syntax as the print and reject
```

Comment:
Either use tt type for `fatal_error` as a constant or just get rid of the underscore and take about it in text in a way that matches the title.